guides/developer/crd-extensions.md

```diff
@@ -70,10 +70,8 @@ public class MyCRDHandler extends KubernetesHandler {
   }
 
   @Override
-  public Class<? extends KubernetesV2CachingAgent> cachingAgentClass() {
-    // Caching agent class for your CRD.
-    // See, e.g., `KubernetesReplicaSetCachingAgent`.
-    return MyCRDCachingAgent.class;
+  public KubernetesV2CachingAgentFactory cachingAgentFactory() {
+    return KubernetesCoreCachingAgent::new;
```

Comment:
Do we maybe want this to be `MyCRDCachingAgent::new` to map onto what was there before?

Reply:
I'm not sure, actually. Do you actually need a custom caching agent for this? When you configure `customResources`, we use the `CustomKubernetesHandlerFactory` to generate one. Maybe we should just do that?

Reply:
I presumed that if people were actually extending clouddriver, it's because they wanted to wrtie their own caching agent (rather than just register additional CRDs via config)---but maybe there's a reason I'm missing for writing this class but re-using an existing caching agent?

Reply:
The primary use case we're working against right now is adding stability logic. Based on the examples I pulled from, it looks like those who are doing this today are just using something like the Core caching agent or `KubernetesUnregisteredCustomResourceCachingAgent::new`.